### ISSUE TITLE
Screen space on mobile, hide header

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/core#7826]
We should use [Headroom.js](http://wicky.nillia.ms/headroom.js/) so the header and controls bar is hidden on scrolling down, but immediately shown on a bit of scrolling up. Important to preserve space on mobile.

(Ported from https://github.com/owncloud/core/pull/7724, cc @owncloud/designers)

**Comment by raghunayyar** [comment]
@jancborchardt this is to be implemented on desktop and mobiles or mobiles only? I can take care of this.

**Comment by jancborchardt** [comment]
Mobile only I would say. And I’m not entirely sure if we need it yet. Maybe check out some issues in 8.2 or so first. :)

**Comment by elpraga** [comment]
I will reply here as I found that a working solutions already exists - at least for my my case. 

I wanted to use Tasks from OC, as I found it to be the best Tasks app I came across.

The only think a user has to do is to choose "Add shortcut to homescreen" from Google Chrome for Android (not sure for other browsers) menu. After that the app (tasks) in my case appears without the address bar.

Might not be a solution for all cases, but surely was for mine. It could be mentioned somewhere so users know that it exists..
